Source/Scene/Scene.js

```diff
@@ -822,6 +822,19 @@ define([
         },
 
         /**
+         * Gets an event that's raised when the terrain provider is changed
+         * @memberof Scene.prototype
+         *
+         * @type {Event}
+         * @readonly
+         */
+        terrainProviderChanged : {
```

Comment:
Since the `terrainProvider` is public on both `Scene` and `Globe` it would be a little weird to have this private on the `Globe`.